examples/02-plot/multi-window.py

```diff
@@ -116,8 +116,9 @@
 
 plotter = pv.Plotter(shape=shape, row_weights=row_weights, col_weights=col_weights, groups=groups)
 
+###############################################################################
 # A grouped subplot can be activated through any of its composing cells using
-# the :func:`pyvista.BasePlotter.subplot` function.
+# the :func:`pyvista.Plotter.subplot` method.
```

Comment:
This block looked like it was supposed to be outside the code blocks. I've added the break header, but we'll have to check the doc build artifact to make sure the result doesn't look weird in some way (I'm mostly concerned by having a `Plotter` created in the first block but shown in the second).

Reply:
The original change broke the doc build so I've made this into a proper comment with no sphinx markup.